elfloader-tool/src/plat/odroidc4/platform_init.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+#include <autoconf.h>
+#include <mode/arm_generic_timer.h>
+
+/* Reset the virtual offset for the platform timer to 0 */
+void platform_init(void)
+{
+    reset_cntvoff();
+}
+
+#if CONFIG_MAX_NUM_NODES > 1
```

Comment:
There is the alias `ENABLE_SMP_SUPPORT` that is also in use, is there a policy when to use what?

Reply:
On a 2nd look, seems this is only used in the kernel, but not in the ELF loader. So ignore that comment above.

Reply:
I think `ENABLE_SMP_SUPPORT` is only defined inside the kernel.

Reply:
It used to be, but then the kernel's private config file was moved into libsel4: https://github.com/seL4/seL4/blob/master/libsel4/include/sel4/config.h#L96

Reply:
Could be a thing for later then: https://github.com/seL4/seL4_tools/issues/96